zingocli/tests/integration_tests.rs

```diff
@@ -3472,4 +3472,8 @@ mod slow {
         assert_eq!(list_before, list_after);
         assert_eq!(witness_before.unwrap(), witness_after.unwrap());
     }
+    #[tokio::test]
+    async fn os_target() {
+        dbg!(LightClient::is_mobile_target());
```

Comment:
do we need to merge this dbg into dev?

Reply:
there are many dbg!s in integration tests.

Reply:
ok  i am of the opinion that dbgs clutter up the console and should be added for debugging and remove for merges into dev. but i suppose dbgs actually being directly inside a test is not so much of an issue for me. also what is this test actually testing, it looks like a unit test not an ingtergration test to me. could you add this to a test module in lightclient/test_features.rs as a unit test and have an assert which checks it returns false (as these tests will not run for mobile archs anyway)

Reply:
actually, could we architect the code so rather than creating a function which returns a boolean for if we are on mobile archs we compile gate so mobile specific code is not compiled for non-mobile and vice-versa? that would probably be safer and also make the binaries smaller

Reply:
oh yeah, maybe its a unit test. i just wanted to be codamn sure that we are compiling integration tests for mobile. and this proved it to be. we dont use the is_mobile_target method anywhere else

Reply:
moved to test_features